get_session_request.php

```diff
@@ -0,0 +1,28 @@
+<?php
+require_once 'config.php';
+
+function get_session_request($contents) {
+    $parsedjson = json_decode($contents, true);
```

Comment:
We need JWTs here and no `unsigned_callback`, because we have one IRMA server at the Foundation's infrastructure and at the moment the IRMA server doesn't support signed and unsigned callbacks to be used differently per requestor. (Or, other solution, we have to change this in the IRMA server).